python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -434,16 +428,18 @@ def _build_policy_network(self, obs, obs_space, action_space):
             action_out = obs
 
         activation = getattr(tf.nn, self.config["actor_hidden_activation"])
-        normalizer_fn = layers.layer_norm if self.config["parameter_noise"] \
-            else None
         for hidden in self.config["actor_hiddens"]:
-            action_out = layers.fully_connected(
-                action_out,
-                num_outputs=hidden,
-                activation_fn=activation,
-                normalizer_fn=normalizer_fn)
-        action_out = layers.fully_connected(
-            action_out, num_outputs=self.dim_actions, activation_fn=None)
+            if self.config["parameter_noise"]:
+                action_out = tf.contrib.layers.fully_connected(
```

Comment:
no more contrib right?

Reply:
https://github.com/tensorflow/tensorflow/issues/26854

Reply:
I kept this since layer norm is nowhere to be found. Guess param noise won't work in tf2.